akka-http-core/src/main/scala/akka/http/impl/engine/ws/WebSocket.scala

```diff
@@ -138,36 +138,27 @@ private[http] object WebSocket {
     val collectMessage: Flow[MessageDataPart, Message, NotUsed] =
       Flow[MessageDataPart]
         .prefixAndTail(1)
-        .mapConcat {
-          // happens if we get a MessageEnd first which creates a new substream but which is then
-          // filtered out by collect in `prepareMessages` below
-          case (Nil, _) ⇒ Nil
-          case (first +: Nil, remaining) ⇒ (first match {
-            case TextMessagePart(text, true) ⇒
```

Comment:
I think we still want to have these cases and create strict messages there, don't we?